runv.go

```diff
@@ -318,31 +352,22 @@ func startVContainer(context *cli.Context) {
 		os.RemoveAll(tmpDir)
 	}()
 
-	if userPod.Resource.Vcpu > 0 {
-		cpu = userPod.Resource.Vcpu
-	}
-
-	if userPod.Resource.Memory > 0 {
-		mem = userPod.Resource.Memory
-	}
-
-	b := &hypervisor.BootConfig{
-		Kernel: kernel,
-		Initrd: initrd,
-		Bios:   "",
-		Cbfs:   "",
-		Vbox:   vbox,
-		CPU:    cpu,
-		Memory: mem,
-	}
-
-	vm := hypervisor.NewVm(vmId, cpu, mem, false, types.VM_KEEP_NONE)
-	err = vm.LaunchOCIVm(b, sock)
+	vm, err := startVm(context, userPod, vmId, sock)
 	if err != nil {
 		fmt.Printf("%s\n", err.Error())
 		return
 	}
 
+	defer func() {
+		Response = vm.StopPod(mypod, "yes")
+
+		if Response.Data == nil {
+			fmt.Printf("StopPod fail: QEMU response data is nil\n")
+			return
+		}
+		fmt.Printf("result: code %d %s\n", Response.Code, Response.Cause)
+	}()
+
```

Comment:
If the error is encountered before the line of `339`, the `defer` function may panic since there is no `mypod` object, right?